src/gbd.h

```diff
@@ -6,4 +6,7 @@
 int
 analyze_gbi (const char *file_name, uint32_t start_addr, bool print_textures, bool print_vertices, bool print_matrices);
 
+static void
+arg_handler (int arg_num);
```

Comment:
This forward declaration should probably go in the C file just above where it needs to be used rather than in the header, since it's not meant to be used outside of the file.